alpha/alpha-spring-cloud-starter-consul/src/main/java/org/apache/servicecomb/pack/alpha/server/discovery/consul/AlphaConsulAutoConfiguration.java

```diff
@@ -114,7 +114,11 @@ public void listenInstanceRegisteredEvent(InstanceRegisteredEvent instanceRegist
             NewService newservice =  new NewService();
             newservice.setName(service.getServiceName());
             newservice.setId(service.getServiceId());
-            List<String> tags = consulDiscoveryProperties.getTags();
+            newservice.setAddress(service.getAddress());
+            newservice.setPort(service.getServicePort());
+            newservice.setMeta(service.getServiceMeta());
+
+            List<String> tags = service.getServiceTags();
```

Comment:
Here you are right, Otherwise, we will lose the default tag, such as secure=false.

With the `List<String> tags = service.getServiceTags();` , we send the following JSON data when we update the consul. 
{
	"ID": "servicecomb-alpha-server-0-0-0-0-e388eb111394660d63c2286c06ee78a6",
	"Name": "servicecomb-alpha-server",
	"Tags": ["alpha-server-host\u003d0.0.0.0", "secure\u003dfalse", "alpha-server-port\u003d8080"],
	"Port": 56701
}